src/main/java/gregtech/common/covers/CoverConveyor.java

```diff
@@ -682,8 +682,12 @@ public void readFromNBT(@NotNull NBTTagCompound tagCompound) {
         this.distributionMode = DistributionMode.VALUES[tagCompound.getInteger("DistributionMode")];
         this.isWorkingAllowed = tagCompound.getBoolean("WorkingAllowed");
         this.manualImportExportMode = ManualImportExportMode.VALUES[tagCompound.getInteger("ManualImportExportMode")];
-        this.itemFilterContainer.deserializeNBT(tagCompound.getCompoundTag("Filter"));
-        this.itemFilterContainer.handleLegacyNBT(tagCompound.getCompoundTag("Filter"));
+        var filterTag = tagCompound.getCompoundTag("Filter");
+        if (filterTag.hasKey("IsBlacklist")) {
```

Comment:
Is this converting the legacy tag into the new format? If so, should `IsBlacklist` be removed from the tag?

Reply:
well, since "IsBlacklist" is written to a different place in the tag, this should only run once. i could remove the tag, but i don't see how that's useful.